src/test/groovy/com/jvm_bloggers/entities/blog/BlogSpec.groovy

```diff
@@ -16,7 +18,7 @@ class BlogSpec extends Specification {
             .author("John Doe")
             .rss("http://john-doe.com/feed/")
             .url("url")
-            .dateAdded(new NowProvider().now())
+            .dateAdded(LocalDateTime.now(NowProvider.DEFAULT_ZONE))
```

Comment:
If we have `ZoneTimeProvider` why use LocalDateTime which is using `NowProvider`? Maybe simply use `ZoneTimeProvider`?
This applies to other places with this `LocalDateTime.now(NowProvider.DEFAULT_ZONE)` call